guides/INSTALLING.md

```diff
@@ -123,6 +130,7 @@ All the services in the [web] and [publishing] journeys, as well as:
 * [dp-hierarchy-builder](https://github.com/ONSdigital/dp-hierarchy-builder)
 * [dp-dimension-search-builder](https://github.com/ONSdigital/dp-dimension-search-builder)
 * [dp-publishing-dataset-controller](https://github.com/ONSdigital/dp-publishing-dataset-controller)
+* [dp-upload-service](https://github.com/ONSdigital/dp-upload-service)
```

Comment:
The upload service is now not just used by datasets but it is also used by homepage editing so may be needed for publishing journeys too. So too the dp-download-service. I'm not sure how to address this, whether to move both up into some common section (with zebedee / api-router etc.) or duplicate (not preferable).

Reply:
As the upload service usage is still wholly within the confines of publishing, I think it's okay to bump that up to the "Publishing journey" list. I think it might be alright to do this with `dp-download-service` as well, as we do mention in the docs to run the entire publishing stack whenever doing any CMD import stuff:

```
All of the services in the web, publishing and CMD journeys need to be run for the full CMD journey to work. This journey includes importing data, publishing it and testing the public journey.
```